### DISCUSSION TITLE
Could not find compatible versions when installing the Type-on-Strap gem

### DISCUSSION THREAD

**Discussion by Zeeshanhayat** [sylhare/Type-on-Strap#459] [Q&A] [answered]
I am getting the following error when running jekyll:

Type-on-Strap> bundle exec jekyll serve
Could not find compatible versions

Because every version of type-on-strap depends on jekyll-paginate ~> 1.1.0
  and jekyll-paginate ~> 1.1.0 could not be found in locally installed gems,
  type-on-strap cannot be used.
So, because Gemfile depends on type-on-strap >= 0,
  version solving has failed.

**Comment by sylhare** [comment, marked as answer]
Hello @Zeeshanhayat did you try to run:

```sh
bundle install
```

Before running `bundle exec jekyll serve` to install all dependencies required. Let me know if this solves your issue.
